### PR TITLE
#1418 #1061

### PR DESCRIPTION
**Pull request by MikhailArkhipov**
Fix command context guid for tool windows
Add toolbar with commands per #1061
Add 'run shiny' and 'stop shiny' commands
Tracking browser window closure is separate task #1427

### CODE REVIEW

File: src/Package/Impl/Repl/Shiny/RunShinyAppCommand.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+using Microsoft.R.Components.InteractiveWorkflow;
+using Microsoft.R.Host.Client;
+using Microsoft.VisualStudio.R.Package.Commands;
+using Microsoft.VisualStudio.R.Packages.R;
+
+namespace Microsoft.VisualStudio.R.Package.Repl.Shiny {
+    internal sealed class RunShinyAppCommand : PackageCommand {
+        private readonly IRInteractiveWorkflow _interactiveWorkflow;
+
+        public static Task RunningTask { get; private set; }
```
Comment (AlexanderSher) [inline src/Package/Impl/Repl/Shiny/RunShinyAppCommand.cs]: Why static?

Comment (MikhailArkhipov) [inline src/Package/Impl/Repl/Shiny/RunShinyAppCommand.cs]: So I can find it from StopShinyAppCommand. Alternative is a bit more complicated: locate menu command service or UI shell and ask for the command status .

Comment (AlexanderSher) [inline src/Package/Impl/Repl/Shiny/RunShinyAppCommand.cs]: Оr you can just define it inside `RInteractiveWorkflow` оr `RInteractiveWorkflowOperations`

Comment (MikhailArkhipov) [inline src/Package/Impl/Repl/Shiny/RunShinyAppCommand.cs]: Kinda specific to Shiny but probably makes sense

Comment (MikhailArkhipov) [inline src/Package/Impl/Repl/Shiny/RunShinyAppCommand.cs]: Done

File: src/R/Components/Impl/InteractiveWorkflow/Implementation/RInteractiveWorkflowOperations.cs [hunk 2]
```diff
@@ -173,6 +174,17 @@ internal class RInteractiveWorkflowOperations : IRInteractiveWorkflowOperations
...
         }
 
+        public void TryRunShinyApp () {
+            _shinyRunningTask = Task.Run(async () => {
+                try {
+                    using (var e = await _workflow.RSession.BeginInteractionAsync()) {
+                        await e.RespondAsync("library(shiny)" + Environment.NewLine + "runApp()" + Environment.NewLine);
+                    }
+                } catch (TaskCanceledException) { } catch (MessageTransportException) { }
+            }).ContinueWith((t) => _shinyRunningTask = null);
```
Comment (AlexanderSher) [inline src/R/Components/Impl/InteractiveWorkflow/Implementation/RInteractiveWorkflowOperations.cs]: you can move `_shinyRunningTask = null` to `finally` block

**Comment by AlexanderSher** [comment]
:+1:

**Comment by int19h** [comment]
:+1:

**Pull request merged**